CMakeLists.txt

```diff
@@ -62,81 +61,29 @@ include_directories(${bredis_SOURCE_DIR}/include ${Boost_INCLUDE_DIRS})
 MESSAGE(STATUS "boost inc: " ${Boost_INCLUDE_DIRS})
 MESSAGE(STATUS "boost libs: " ${Boost_LIBRARIES})
 
-add_library(catch_lib t/tests-main.cpp)
+# add_library(catch_lib t/tests-main.cpp)
 
 if(WIN32)
     set(Boost_USE_MULTITHREADED ON)
-    set(LINK_DEPENDENCIES ${Boost_LIBRARIES} catch_lib)
+    set(LINK_DEPENDENCIES ${Boost_LIBRARIES})
     add_definitions(-DBOOST_ALL_DYN_LINK -DBOOST_THREAD_USE_LIB -D_WIN32_WINNT=0x0501)
 else()
-    set(LINK_DEPENDENCIES pthread ${Boost_LIBRARIES} catch_lib)
+    #add_definitions(-Wall -Wextra -pedantic -Werror)
```

Comment:
I think this line needs to be deleted. because it is not part of CMake file, also it is not a useful comment. By the way there are some lines with same situation in this file.